version4.go

```diff
@@ -14,6 +14,14 @@ func New() UUID {
 	return Must(NewRandom())
 }
 
+// NewString creates a new random UUID and returns it as a string or panics.
+// NewString is equivalent to the expression
+//
+//    uuid.Must(uuid.NewRandom()).String()
+func NewString() string {
+	return New().String()
```

Comment:
Small nit, but might as well get rid of one call and say return Must(NewRandom()).String().

I guess basically swap lines 20 and 22 :-)

Reply:
Sounds good